scripts/lib/CIME/case/case.py

```diff
@@ -773,11 +773,10 @@ def configure(self, compset_name, grid_name, machine_name=None,
             compset_name, files)
 
         self._components = self.get_compset_components()
+
         #--------------------------------------------
         # grid
         #--------------------------------------------
-        if user_grid is True and gridfile is not None:
```

Comment:
Was this deletion intentional? It looks unrelated to the other changes in this PR. If it was intentional, can you please describe why you deleted it?

Reply:
There should be 2 lines deleted:
-        if user_grid is True and gridfile is not None:
-            self.set_value("GRIDS_SPEC_FILE", gridfile)
because they generated an error when I was implementing
a high resolution SST grid (#1798) and @jedwards4b 
suggested removing those lines.  I need the hi-res in my 
test case, but I suppose for the purpose of this PR I should
use a different test case and keep those 2 lines.
Should I add them back into my branch and push it again,
or go back to an earlier commit and start over with a new,
cleaner PR?

Reply:
Okay, thanks for the clarification. In an ideal world, this would be in a separate PR, but I'm not going to be a stickler there. If you want to keep it as part of this PR, it's fine with me if you just revert that change on this branch. However, please edit the initial PR comment to document this change (ideally it would also be associated with an issue, but again, I won't be a stickler).

Reply:
I edited the initial comment to include a brief description 
of the deletion from case.py.  Thanks for your flexibility.